<ui version="4.0" >
 <class>Raw_image_dialog</class>
 <widget class="QDialog" name="Raw_image_dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Open raw image</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Image &amp;value type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="2" column="0" >
       <widget class="QRadioButton" name="short_bt" >
        <property name="text" >
         <string>Short (16 bits)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QRadioButton" name="float_bt" >
        <property name="text" >
         <string>Float</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="int_bt" >
        <property name="text" >
         <string>Int (32 bits)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QRadioButton" name="radioButton_6" >
        <property name="text" >
         <string>Double</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QCheckBox" name="signed_bt" >
        <property name="text" >
         <string>Signed</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Image dimensions</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_dim" >
          <property name="text" >
           <string>&amp;Dimensions:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>dim_x</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_spacing" >
          <property name="text" >
           <string>&amp;Spacing:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>spacing_x</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QSpinBox" name="dim_x" >
          <property name="prefix" >
           <string>x: </string>
          </property>
          <property name="maximum" >
           <number>100000000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QSpinBox" name="dim_y" >
          <property name="prefix" >
           <string>y: </string>
          </property>
          <property name="maximum" >
           <number>100000000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QSpinBox" name="dim_z" >
          <property name="prefix" >
           <string>z: </string>
          </property>
          <property name="maximum" >
           <number>100000000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QDoubleSpinBox" name="spacing_x" >
          <property name="prefix" >
           <string>vx: </string>
          </property>
          <property name="decimals" >
           <number>3</number>
          </property>
          <property name="value" >
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QDoubleSpinBox" name="spacing_y" >
          <property name="prefix" >
           <string>vy: </string>
          </property>
          <property name="decimals" >
           <number>3</number>
          </property>
          <property name="value" >
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QDoubleSpinBox" name="spacing_z" >
          <property name="prefix" >
           <string>vz: </string>
          </property>
          <property name="decimals" >
           <number>3</number>
          </property>
          <property name="value" >
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="offset_label" >
          <property name="text" >
           <string>&amp;Offset:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>offset</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QSpinBox" name="offset" >
          <property name="suffix" >
           <string> bytes</string>
          </property>
          <property name="maximum" >
           <number>999999999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Open</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>int_bt</tabstop>
  <tabstop>short_bt</tabstop>
  <tabstop>signed_bt</tabstop>
  <tabstop>float_bt</tabstop>
  <tabstop>radioButton_6</tabstop>
  <tabstop>dim_x</tabstop>
  <tabstop>dim_y</tabstop>
  <tabstop>dim_z</tabstop>
  <tabstop>spacing_x</tabstop>
  <tabstop>spacing_y</tabstop>
  <tabstop>spacing_z</tabstop>
  <tabstop>offset</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>float_bt</sender>
   <signal>toggled(bool)</signal>
   <receiver>signed_bt</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>47</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel" >
     <x>257</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_6</sender>
   <signal>toggled(bool)</signal>
   <receiver>signed_bt</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>281</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel" >
     <x>290</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Raw_image_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>191</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel" >
     <x>183</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Raw_image_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>253</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel" >
     <x>252</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
